app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentContentCardTest.kt

```diff
@@ -0,0 +1,33 @@
+package org.oppia.app.player.state
+
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.hasDescendant
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.home.HomeActivity
+import org.oppia.app.recyclerview.RecyclerViewMatcher.Companion.atPosition
+import org.oppia.app.testing.ContentCardTestActivity
+
+// TODO(#277): Add tests for image parsing.
+
+/** Tests for [VIEW_TYPE_CONTENT]. */
+@RunWith(AndroidJUnit4::class)
+class StateFragmentContentCardTest {
+
+  @Test
+  fun testContentCard_forDemoExploration_withCustomOppiaTags_displaysParsedHtml() {
```

Comment:
As per my conversation with @BenHenning you need to shift this test-case to `StateFragmentTest` file.

Also, you will need to follow this same approach to #276 also, meaning everything will be controlled by a TestActivity which calls StateFragment and the test case will be in StateFragmentTest.

Reply:
Do I need to load exploration in StateFragmentTest file? Because now it is loading in `ContentCardTestActivity`.

Reply:
You just need to shift this test directly to `StateFragmentTest` and it should work. The launch activity in that test will still be `ContentCardTestActivity `

Reply:
Then its done.